.github/workflows/uffizzi-build.yaml

```diff
@@ -0,0 +1,87 @@
+name: Build PR Image
+on:
+  pull_request:
+    types: [opened,synchronize,reopened,closed]
```

Comment:
I agree with @KKoukiou about not triggering this for every PR run. And this is especially not needed on "closed".

We used to trigger this with a magic comment `/livetest`, see the [old workflow](https://github.com/cockpit-project/cockpit/blob/266/.github/workflows/livetest.yml). Something similar could be done here.

Reply:
Ah, nevermind, "closed" is used for cleaning up. But what about cleaning up old instances from force pushes? Don't they pile up?

Reply:
Sure, you can/should use the default GITHUB_TOKEN for that. However, I'm happy to work on this part, I've done it before and it's not hard. So it's fine for you to leave the trigger alone in this PR.